.github/workflows/acceptance-tests-on-approval.yml

```diff
@@ -0,0 +1,81 @@
+
+# This workflow runs when a PR review is submitted and marked as “approved”.
+name: acceptance-tests-on-approval
+
+on:
+  pull_request_review:
```

Comment:
This workflow needs docker creds and will fire only when a team member approves a PR